ansible/inventories/sandbox/group_vars/inventory-next/vars.yml

```diff
@@ -0,0 +1,53 @@
+---
+# In sandbox, we use python3 on bionic web hosts
+# https://github.com/GSA/datagov-infrastructure-modules/issues/32
+ansible_python_interpreter: /usr/bin/python3
+
+datagov_service: inventory
+app_type: inventory-next
+inventory_db_name: inventory_db_next
+inventory_python_home: "{{ inventory_ckan_python_home }}"
+datagov_in_service: false
+
+
+# Inventory Datapusher
+inventory_datastore_ro_user: datastore_ro
+inventory_datastore_user: datastore
+
+# project path
+project_source_rollback_path: /usr/lib/ckan-rollback
+project_source_new_code_path: /usr/lib/ckan-new
+current_source_symlink: /usr/lib/ckan
+project_index_path: /usr/lib/ckan
+
+# db credentials
+inventory_ckan_db_host: "{{ inventory_next_postgresql_login_host }}"
+inventory_db_user: "{{ inventory_next_postgresql_login_user }}"
+inventory_db_pass: "{{ inventory_next_postgresql_login_password }}"
+datastore_db_server: "{{ inventory_next_postgresql_login_host }}"
+inventory_datapusher_db_pass: "{{ inventory_next_datapusher_db_pass }}"
+inventory_datastore_pass: "{{ inventory_next_datastore_pass }}"
+inventory_datastore_ro_pass: "{{ inventory_next_datastore_ro_pass }}"
+
+# other
+inventory_ckan_bucket_name: "{{ inventory_next_ckan_s3_bucket_name }}"
+inventory_ckan_bucket_prefix: "{{ inventory_next_ckan_s3_bucket_prefix }}"
+ckan_site_domain: "{{ inventory_next_ckan_service_url }}"
+
+inventory_app_repo_branch: inventory_ckan_2.8
+
+# TODO is this for all databases (inventory, datastore, datapusher)? Don't think we need this
+# postgresql_enable_extensions:
+#   - postgis
```

Comment:
Yeah, you can remove this since we're not using datagov-deploy-postgresql (although it would make db initialization much easier)